code-challenges.test.js

```diff
@@ -16,35 +20,117 @@
 // a) Create a test with an expect statement using the variable provided. 
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+     describe("shuffle",() => {
+         it("removes the first item from the array and shuffles the remaining content.",() => {
+             const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+                         // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+        expect(shuffle).toEqual(expect.anything());
+
+
+             const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+                         // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+        expect(shuffle).toEqual(expect.anything());
+            
+
+         })
+    })
+
+                          //  *****TEST RESULTS: Failed -----------> ReferenceError: shuffle is not defined
+                                         //Test Suites: 1 failed, 1 total
+                                         //Tests:       1 failed, 1 total
 
 
 // b) Create the function that makes the test pass.
+        ///// INPUT: colors1, colors1
+        /////OUTPUT:["blue", "green", "yellow", "pink"] ; ["indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+
+                                ///Setting up the Jest testing framework, I discover a jest method call anything that will leave
+                        /// it relatively open to any test results. This is is help when I use the slice method to take off the 
+                        // first element in each array. I get a good failing test with my variable |shuffle|. Creating my function I    
+                        // plan to return the arrays by slicing the first element in each of the color1, color2 arrays. The problem is 
+                        // that the terminal output is [Function: shuffle]. it knows it's a function but for some reason my code is not running.
+                        // My function may not be invoked?
+
+         const shuffle = (arr) => {
+            return arr(colors1.slice(1))(colors2.slice(1))
```

Comment:
Make sure to pay attention to the intent or expected output of the function should be.  For this problem, we want to randomize the array after removing the first element.  Slice seems like a good start to remove the first instance, however, how can you randomize the rest of the array?